geom/geom/src/TGeoElement.cxx

```diff
@@ -151,6 +151,20 @@ TGeoElement::TGeoElement(const char *name, const char *title, Int_t z, Int_t n,
    fAbundances = NULL;
    ComputeDerivedQuantities();
 }
+
+////////////////////////////////////////////////////////////////////////////////
+/// destructor
+
+TGeoElement::~TGeoElement()
+{
+   if (fIsotopes) {
+      // fIsotopes->Delete(); // Andrei, check if isotopes shhould be deleted as well
+      delete fIsotopes;
+   }
```

Comment:
Every isotope is added to the element table list of isotopes and deleted in its destructor, so no, they should not be deleted here. You can replace the whole block with just: delete fIsotopes; (works also if fIsotopes is nullptr)